source/h2_connection.c

```diff
@@ -735,16 +771,66 @@ static int s_handler_process_read_message(
     struct aws_channel_handler *handler,
     struct aws_channel_slot *slot,
     struct aws_io_message *message) {
-
-    (void)handler;
     (void)slot;
-    (void)message;
+    struct aws_h2_connection *connection = handler->impl;
+    int err;
+
+    CONNECTION_LOGF(
+        TRACE,
+        connection,
+        "id=%p: H2 connection Begin processing message of size %zu.",
+        (void *)&connection->base,
+        message->message_data.len);
```

Comment:
the `CONNECTION_LOGF` macro automatically does the "id=%p" stuff, so you can just write this as:
```
CONNECTION_LOGF(TRACE, connection, "Begin processing message of size %zu.", message->message_data.len);
```

likewise for all the other logging statements here...